mysql-test/suite/galera/t/galera_retry_applying2.test

```diff
@@ -0,0 +1,40 @@
+#
```

Comment:
The common practice in mtr testing is to bundle several test cases in same test file, this should yield faster testing. As galera_retry_applying and galera_retry_applying2 quite similar test cases, please accommodate both in galera_retry_applying.test and  remove the galera_retry_applying2.test and result as well

Reply:
Fixed as suggested.